algorithms/estonian/stemmer.tt

```diff
@@ -0,0 +1,479 @@
+        <DD>replace by <B><I>luge</I></B>
+    <DT><B><I>põen  &nbsp;  põeb  &nbsp;  põed  &nbsp;  põeme  &nbsp;  põete &nbsp;  põevad</I></B>
+        <DD>replace by <B><I>põde</I></B>
+    <DT><B><I>põeksin  &nbsp;  põeks  &nbsp;  põeksid  &nbsp;  põeksime  &nbsp;  põeksite</I></B>
+        <DD>replace by <B><I>põde</I></B>
+    <DT><B><I>laon  &nbsp;  laob  &nbsp;  laod  &nbsp;  laome  &nbsp;  laote  &nbsp;  laovad</I></B>
+        <DD>replace by <B><I>ladu</I></B>
+    <DT><B><I>laoksin  &nbsp;  laoks  &nbsp;  laoksid  &nbsp;  laoksime  &nbsp;  laoksite</I></B>
+        <DD>replace by <B><I>ladu</I></B>
+    <DT><B><I>teeksin  &nbsp;  teeks  &nbsp;  teeksid  &nbsp;  teeksime  &nbsp;  teeksite</I></B>
+        <DD>replace by <B><I>tegi</I></B>
+    <DT><B><I>teen  &nbsp;  teeb  &nbsp;  teed  &nbsp;  teeme  &nbsp;  teete  &nbsp;  teevad</I></B>
+        <DD>replace by <B><I>tegi</I></B>
+    <DT><B><I>tehakse  &nbsp;  tehti  &nbsp;  tegema  &nbsp;  tegemata  &nbsp;  teha</I></B>
+        <DD>replace by <B><I>tegi</I></B>
+    <DT><B><I>näen  &nbsp;  näeb  &nbsp;  näed  &nbsp;  näeme  &nbsp;  näete  &nbsp;  näevad</I></B>
+        <DD>replace by <B><I>nägi</I></B>
+    <DT><B><I>näeksin  &nbsp;  näeks  &nbsp;  näeksid  &nbsp;  näeksime  &nbsp;  näeksite</I></B>
+        <DD>replace by <B><I>nägi</I></B>
+    <DT><B><I>nähakse  &nbsp;  nähti  &nbsp;  näha  &nbsp;  nägema  &nbsp;  nägemata</I></B>
+        <DD>replace by <B><I>nägi</I></B>
+</DL>
+</DL>
+
+
+<p>
+Step 1: emphasis
+</p>
+
+<DL><DD>
+    Search for the longest among the following suffixes in <I>R1</I>, and perform the action indicated
+<BR><BR>
+<DL>
+        <DD>Test if there’s at least 4 letters before <I>R1</I> region. If so, continue this step
+    <DT><B><I>gi</I></B>
+        <DD>if preceded by <B><I>GI</I></B> and <B>not <I>LONGV</I></B>, delete
+    <DT><B><I>ki</I></B>
+        <DD>if preceded by <B>är</B>, replace it with <B>k</B>. If it wasn't replaced with <B>k</B> and is preceded by <B><I>KI</I></B>, delete
+</DL>
+</DL>
+
+<p>
+Step 2: verb
+</p>
+<DL><DD>
+    Search for the longest among the following suffixes in <I>R1</I>, and perform the
+    action indicated
+</p>
+
+<DL>
+    <DT><B><I>nuksin  &nbsp;  nuksime  &nbsp;  nuksid  &nbsp;  nuksite</I></B>
+		<DD>delete
+    <DT><B><I>ksin  &nbsp;  ksid  &nbsp;  ksime  &nbsp;  ksite</I></B>
+		<DD>delete
+    <DT><B><I>mata</I></B>
+		<DD>delete
+    <DT><B><I>takse  &nbsp;  dakse</I></B>
+		<DD>delete
+    <DT><B><I>taks  &nbsp;  daks</I></B>
+		<DD>delete
+    <DT><B><I>akse</I></B>
+		<DD>delete
+    <DT><B><I>sime</I></B>
+		<DD>delete
+    <DT><B><I>site</I></B>
+		<DD>delete
+    <DT><B><I>sin</I></B>
+		<DD>delete
+    <DT><B><I>me preceded by </I>V1</B>
+		<DD>delete
+    <DT><B><I>da preceded by </I>V1</B>
+		<DD>delete
+    <DT><B><I>n preceded by </I>V1</B>
+		<DD>delete
+    <DT><B><I>b preceded by </I>V1</B>
+		<DD>delete
+		
+		<DD>set is_verb
```

Comment:
This is in the wrong place since it's set if any of the above rules fire, and `is_verb` isn't mentioned elsewhere in this description.  It's also covered by the prose instructions "If nothing was changed in step 2, do [...]. If something was changed in step 2, do [...]" so removing.